go.mod

```diff
@@ -1,41 +1,31 @@
 module github.com/kubeflow/common
 
-go 1.11
+go 1.13
 
 require (
-	github.com/docker/distribution v2.7.1+incompatible // indirect
-	github.com/emicklei/go-restful v2.9.3+incompatible // indirect
 	github.com/ghodss/yaml v1.0.0 // indirect
-	github.com/go-openapi/spec v0.19.0
-	github.com/gogo/protobuf v1.2.1 // indirect
+	github.com/go-openapi/spec v0.19.2
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
-	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/google/btree v1.0.0 // indirect
 	github.com/googleapis/gnostic v0.2.0 // indirect
-	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
-	github.com/kubernetes-sigs/kube-batch v0.4.2
-	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/sirupsen/logrus v1.4.1
-	github.com/spf13/pflag v1.0.3 // indirect
-	github.com/stretchr/testify v1.2.2
-	golang.org/x/crypto v0.0.0-20190325154230-a5d413f7728c // indirect
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	github.com/stretchr/testify v1.3.0
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/square/go-jose.v2 v2.3.0 // indirect
 	k8s.io/api v0.15.10
-	k8s.io/apimachinery v0.15.10
-	k8s.io/client-go v0.15.10
+	k8s.io/apimachinery v0.16.9-beta.0
+	k8s.io/client-go v0.16.9-beta.0
 	k8s.io/code-generator v0.15.10
 	k8s.io/kube-openapi v0.0.0-20190228160746-b3a7cee44a30
-	k8s.io/kubernetes v1.15.10
+	k8s.io/kubernetes v1.18.0
```

Comment:
Can we try to minimize the dependency changes? Another thing is `k8s.io/Kubernetes` version doesn't match with apimachinery and client-go

Reply:
+1 to keep current version :) Is there any reason for that?

Reply:
Actually we import 1.15.10, you can see the replace below. This is autogenrerated

Reply:
Em. It must be some libraries requires high version?  I notice volcano  go.mod requires 1.16.9 https://github.com/volcano-sh/volcano/blob/9c314bd9e262413be6a902ae1dc6dbe133732ca5/go.mod#L49-L54

which matches apimachinery and client-go version here. `k8s.io/kubernetes` must be from other libraries.